samples/powerpoint/tags/tags.yaml

```diff
@@ -139,6 +145,44 @@ script:
           });
         }
 
+        async function setup() {
+          await PowerPoint.run(async (context) => {
+            // Adds two new slides with some content.
+            const slideCountResult = context.presentation.slides.getCount();
```

Comment:
Can we rename this variable to "startingSlideCount" or something? I got a little confused later when it looked like we were going to have out of bounds exceptions (slideCount + 1).